recipe/meta.yaml

```diff
@@ -39,12 +39,14 @@ requirements:
     - pybind11 >=2.2.3
     - adios >=1.13.1  # [unix]
     - hdf5  >=1.8.13
+    - blosc
```

Comment:
those three are probably not needed here. Underlying issue is: https://github.com/conda-forge/adios-feedstock/pull/12#discussion_r247309803